subsys/net/ip/l2/ieee802154/ieee802154.c

```diff
@@ -32,53 +32,13 @@
 
 #if 0
```

Comment:
I know it's outside the patch, but having commented-out code like this is not good.
Make this in terms of some macro IMO

Reply:
A Kconfig option will do yes.